scout/server/blueprints/variants/controllers.py

```diff
@@ -1595,15 +1594,16 @@ def update_form_hgnc_symbols(store, case_obj, form):
                 hgnc_gene_caption = store.hgnc_gene_caption(int(hgnc_symbol), genome_build)
                 if hgnc_gene_caption is None:
                     not_found_ids.append(hgnc_symbol)
-                else:
-                    hgnc_symbols.append(hgnc_gene_caption.get("hgnc_symbol", hgnc_symbol))
-            else:
-                hgnc_symbols.append(hgnc_symbol)
+                    continue
+                hgnc_symbols.append(hgnc_gene_caption.get("hgnc_symbol", hgnc_symbol))
+                continue
+
+            hgnc_symbols.append(hgnc_symbol)
 
     # add HPO genes to list, if they were missing
     if "hpo" in form.data.get("gene_panels", []):
         hpo_symbols = list(
-            set(term_obj["hgnc_symbol"] for term_obj in case_obj["dynamic_gene_list"])
+            set(term_obj["hgnc_symbol"] for term_obj in case_obj.get("dynamic_gene_list"))
```

Comment:
Change to `case_obj.get("dynamic_gene_list", [])`? 

This would avoid `TypeError: 'NoneType' object is not iterable` if the get-function returns None.

Reply:
Right! Fixed!